File: atcoder/375/A-seat.py

# 入力の取得
N = int(input())
S = input()

count = 0
# 1 から N-2 の範囲で条件を満たすかチェック
for i in range(1, N - 1):
    if S[i - 1] == '#' and S[i] == '.' and S[i + 1] == '#':
        count += 1

# 結果の出力
print(count)

File: atcoder/375/A-seat.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;

    char S[n];
    for (int i = 0; i < n; i++)
    {
        cin >> S[i];
    }

    int count = 0;
    for (int i = 0; i < n - 2; i++)
    {
        if (S[i] == '#' && S[i + 1] == '.' && S[i + 2] == '#')
        {
            count++;
        }
    }

    cout << count;

    return 0;
}